mon-pix/app/components/signin-form.js

```diff
@@ -26,6 +26,10 @@ export default class SigninForm extends Component {
     return this.url.homeUrl;
   }
 
+  get displayPoleEmploiButton() {
```

Comment:
Suggestion : Renommer la fonction par rapport au `french domain`.

Reply:
Ou bien ajouter ici `this.isPoleEmploiEnabled`, pour utiliser simplement dans le `hbs` que `this.displayPoleEmploiButton`.

Reply:
A en discuter en recap tech ( suppression du featureToggle de PE )

Reply:
J'ai réuni le this.isPoleEmploiEnabled dans mon get displayPoleEmploiButton